doom-modeline-segments.el

```diff
@@ -846,19 +868,33 @@ mouse-2: Show help for minor mode")
               (pcase status
                 ('finished  (when flycheck-current-errors
                               (let-alist (doom-modeline--flycheck-count-errors)
-                                (if doom-modeline-check-simple-format
-                                    (doom-modeline-check-text
-                                     (number-to-string (+ .error .warning .info))
-                                     (cond ((> .error 0) 'doom-modeline-urgent)
-                                           ((> .warning 0) 'doom-modeline-warning)
-                                           (t 'doom-modeline-info)))
-                                  (format "%s/%s/%s"
-                                          (doom-modeline-check-text (number-to-string .error)
-                                                                      'doom-modeline-urgent)
-                                          (doom-modeline-check-text (number-to-string .warning)
-                                                                      'doom-modeline-warning)
-                                          (doom-modeline-check-text (number-to-string .info)
-                                                                      'doom-modeline-info))))))
+                                (pcase doom-modeline-check-simple-format
+                                  ('t (doom-modeline-check-text (number-to-string (+ .error .warning .info))
+                                                                (cond ((> .error 0) 'doom-modeline-urgent)
+                                                                      ((> .warning 0) 'doom-modeline-warning)
+                                                                      (t 'doom-modeline-info))))
+                                  ('nil (format "%s/%s/%s"
+                                                (doom-modeline-check-text (number-to-string .error)
```

Comment:
refactor: the texts are the same below. Maybe should use `let` to simplify.

Reply:
Done